tests/debuginfo/skip_second_statement_si_collapse.rs

```diff
@@ -0,0 +1,50 @@
+// ignore-lldb
+#![feature(collapse_debuginfo)]
```

Comment:
No, the problem is exactly that just enabling feature, without adding attributes, breaks this test in the current master. Because println macro don't have collapse attribute.